### DISCUSSION TITLE
connection refused

### DISCUSSION THREAD

**Discussion by aalmazyad** [python-smpplib/python-smpplib#155] [Q&A] [answered]
I get this error I'm using port 2775

```
Traceback (most recent call last):
  File "gateway.py", line 22, in <module>
    client.connect()
  File "/usr/lib/python2.6/site-packages/python_smpplib-1.0.1-py2.6.egg/smpplib/client.py", line 108, in connect
    raise exceptions.ConnectionError("Connection refused")
smpplib.exceptions.ConnectionError: Connection refused
```

what can I do to solve it ?

**Comment by code-of-kpp** [comment]
Can you connect to your server manually e.g. via telnet?

**Comment by aalmazyad** [comment]
Yes I can via ssh or telnet port 22, but when I do telnet localhost 2775 I get connection refused.

**Comment by aalmazyad** [comment]
How do I enable the service or are there requirements that I need to install ?

**Comment by code-of-kpp** [comment]
What do you mean by "via SSH yes"?
It is clearly not a problem with library: you should get manual connection
work (why do you connect to localist?)

**Comment by aalmazyad** [comment]
Ignore the ssh thing
client = smpplib.client.Client('example.com', SOMEPORTNUMBER)
should I fill these with my information or with my service provider info ? 

client = smpplib.client.Client('localhost',2775)

**Comment by code-of-kpp** [comment, marked as answer]
Yes, you should fill it with correct values.
